src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -144,9 +144,24 @@ private bool TryAuthenticateFromCache(
 		Uri orgUrl,
 		string user,
 		bool nonInteractive,
-		bool experimentalBypassBrowserSecurity
+		bool bypassBrowserSecurity
 	) {
-		await using IBrowser? browser = await Browser.LaunchBrowserAsync( experimentalBypassBrowserSecurity );
+
+		bool hasElevatedPermissions = UserPrivileges.HasElevatedPermissions();
+		if( hasElevatedPermissions && !bypassBrowserSecurity ) {
+			consoleWriter.WriteWarning( $"""
+				BMX is being run with elevated privileges and is unable to automatically sign in to Okta.
```

Comment:
Not in scope, but - some warnings messages are prefixed with "WARNING:" while others are not.
Would be nice to be consistent on this.
I lean towards no prefix, because we already use a different colour.

Reply:
Hmm yea you're right. I started with including the prefix since they were in the bmx config provider class but have been inconsistent. I see we use `WriteWarning` in other places though without it. I think I'll remove the prefix for this class